docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,424 @@ HTTP Response
       "status": 200
     }
 
+
+Creating a new User
```

Comment:
```suggestion
Creating a New User
```
Is there a specific reason why "new" is not capitalized? I cannot tell what the rule is, it looks like all verbs which are not articles are capitalized in titles.

Reply:
"User" is a domain element, "new" is colloquial here? 🤷

Reply:
"Proper" English title case would capitalize "new" here; pretty much everything except articles. We're not very strict about title case through the rest of our docs though so 🤷

Reply:
In German even within titles we do not capitalize everything. The usual rules apply there like for any other normal text (everything not capitalized except names and nouns etc.).

Overall I just noticed that I accidentally applied the German rules there :facepalm: To me correct English would have been `Creating a new user`.

Reply:
> To me correct English would have been "Creating a new user".

Outside of a title, yes. Typography rules vary per language; the traditional rule in English for titles in books and journals and stuff is to capitalize everything except articles (unless the title starts with an article, in which case the first letter is still capitalized). So, like, "To the Moon We Go".

Anecdotally, the French rule for titles is to capitalize every word until the first non-article word, roughly: "To The Moon we go".

There's no real logic to these, it's just what people settled on for whatever reason.